### ISSUE TITLE
Shouldn't require/execute resolvers.js when deploying the built in resolvers lambda

### ISSUE THREAD

**Issue by hypexr** [serverless-components/graphql#12]
Unexpected errors can occur when resolvers.js is loaded as part of the deploy process.

For example, it's common to create things like DB connections outside of the handler so that they persist as long as the lambda is running without being recreated on each call.  Another thing that I'm experiencing is that I instrument my lambdas with aws xray and that loads and instruments http which can errors.

If this require was replaced with a fs read file and it was parsed differently it would still be possible to determine the keys under Query and Mutation.

**Comment by eahefnawy** [comment]
how could it be parsed differently? AST? 🤔

**Comment by hypexr** [comment]
Woah, never heard of AST before, but looks like a perfect fit.

**Comment by eahefnawy** [comment]
Using AST is low-level and complicated, so I don't think it'd be worth it for this use case. I was hoping you could share what you mean by "parsing differently" because AST is the only thing I could think of? 😅

**Comment by hypexr** [comment]
Right, I don't know of a better way, but it does seem like AST is appropriate for this problem.  Otherwise it should be specified in a configuration file.
